lib/css/modules/_nightMode.scss

```diff
@@ -24,12 +24,21 @@ $title-link-visited-color: hsl(0, 0%, 65%);
 	.modactionlisting table *,
 	.side .recommend-box .rec-item,
 	.crosspost-preview,
-	.crosspost-thing-preview,
-	.link {
+	.crosspost-thing-preview {
 		background-color: $background-color;
 		border-color: $border-color;
 	}
 
+	.link,
+	.helpcenter-form .section-content {
+		background-color: $background-color;
+		border: none;
```

Comment:
Is `border: none;` doing anything useful here? If not, I think we might as well forego it as to not break custom styles which has specified a border.

Reply:
A weird yellow border existed on the element, could potentially leave it in. Let me revisit and and check it again.

Reply:
Removed in latest commit.